packages/offvis.vm/offvis.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>offvis.vm</id>
+    <version>1.0</version>
+    <authors>Microsoft</authors>
+    <description>The Microsoft Office Visualization Tool (OffVis) is a tool from Microsoft that helps understanding the Microsoft Office binary file format in order to deconstruct .doc-, .xls- and .ppt-based targeted attacks.</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="dotnet3.5" />
```

Comment:
do we need this concrete version of dotnet? We should try to use the same version in all tools if possible. We have introduced metapackages for some dependencies to avoid problems: https://github.com/mandiant/VM-Packages/issues/673 So thinking if we should do something like this for dotnet (assuming we don't need a concrete version or that at least we can share the major vesion with other packages)

Reply:
I'm afraid that the concrete version is needed.

![image](https://github.com/mandiant/VM-Packages/assets/43878201/a285ed0d-7c67-4f6d-8114-09e116874b43)

Reply:
I am now remembering, we are also installing this concrete version in FLARE-VM https://github.com/mandiant/flare-vm/blob/main/config.xml#L28 because otherwise some all .NET binaries don't run

Reply:
Then, I think it's redundant here, removed.

Reply:
Rollbacked.